### ISSUE TITLE
Initial box download very slow

### ISSUE THREAD

**Issue by plowman** [hashicorp/vagrant#11536]
Trying to download the "ubuntu/focal64" box with a plain Vagrantfile and the download speeds are unusably slow right now, like ~7kB/second, 18 hours to download the box:

![Screen Shot 2020-04-21 at 12 57 35 PM](https://user-images.githubusercontent.com/148066/79908756-a604f380-83d0-11ea-81e2-fb2993e744b8.png)

Everything else is fast on my network so I was wondering if this a known issue on your side?

Also, has the Vagrant project given much thought to also distributing boxes via torrent, similar to how [Ubuntu does with releases](https://releases.ubuntu.com/20.04/)? That would at least help unblock people if Vagrant Cloud has network issues/congestion/etc.

### Vagrant version
Vagrant 2.2.7

### Host operating system
macOS Catalina 10.15.4 (19E266)

### Guest operating system
ubuntu/focal64
v20200420.0.0

### Vagrantfile
```ruby
# -*- mode: ruby -*-
# vi: set ft=ruby :

Vagrant.configure("2") do |config|
  config.vm.box = "ubuntu/focal64"
end
```

### Expected behavior
Download is almost always very fast, e.g. ~10 minutes to download it all

### Actual behavior
Currently, expected download time is 18 hours from now.

### Steps to reproduce
1. run `vagrant init ubuntu/focal64 --box-version 20200420.0.0`
2. run `vagrant up`
3. wait

### References
This seems to occur periodically. The last time was March 2018:
- GH-9580 (Closed as duplicate of GH-8434)

### In conclusion
Thanks for your help!

**Comment by briancain** [comment]
Hey there @plowman -

The box you are downloading is not actually hosted directly on Vagrant Cloud. It's externally hosted by Ubuntu: https://app.vagrantup.com/ubuntu/boxes/focal64/versions/20200420.0.0

> Externally hosted (cloud-images.ubuntu.com)

Also from the output from Vagrant in the image shared:

> Download redirected to host: cloud-images.ubuntu.com

Sorry that you are getting slow downloads, but it is with Ubuntus servers, not Vagrant Cloud! This is a fairly common issue, as mentioned in the referenced issues. Unfortunately we have no control over their servers. If it's too slow, I recommend taking a look at the bento boxes, as they are reliable and are hosted on Vagrant Cloud. Thanks!

**Comment by plowman** [comment]
Aha, that is helpful. Thanks, @briancain!
